src/Common/recorder/ReaperProjectGenerator.h

```diff
@@ -8,6 +8,14 @@ class ReaperProjectGenerator : public JamMetadataWriter
 {
 public:
     void write(const Jam &jam) override;
+    inline QString getWriterId() const override
+    {
+        return "ReaperProjectGenerator";
+    }
+    inline QString getWriterName() const override // Localised
+    {
+        return "Generate Reaper Project file (RPP)";
```

Comment:
Looks like I need to use `QCoreApplication::translate` as I've no handy `QObject` to get `::tr` from.  (Or should I inherit from QObject?)